recipes/pymultinest/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: pymultinest
+  version: "2.6"
+
+source:
+  fn: pymultinest-2.6.tar.gz
+  url: https://pypi.python.org/packages/c2/fc/77bdd8356068125bba0a5c07bfa2b4a3f492bff614aeae783bc1a566d699/pymultinest-2.6.tar.gz
+  md5: d9e5eee6abd3784056ca5ccc1f05cab9
+
+build:
+   number: 1
+   skip: true  # [win]
+   skip: true  # [py3k]
+
+requirements:
+
+  build:
+    - toolchain
+    - python x.x
+    - setuptools
+    - multinest
+    - numpy
+    - scipy
+    - matplotlib
+    - pytest
+
+  run:
+    - python x.x
+    - multinest
+    - numpy
+    - scipy
+    - matplotlib
+    - pytest
+
+test:
+  imports:
```

Comment:
I think there should also be import tests for `pypolychord` and `pycuba`

Reply:
`pypolychord` and `pycuba` require two other binary libraries that are not in conda-forge (and I am not sure they can be, because of their licenses, but I am not an expert).

This package is supposed to contain only the wrapper to Multinest, the wrappers for PolyChord and Cuba will be provided in other packages if possible and needed.